### PR TITLE
added HinglishEval in project section

### PR DESCRIPTION
**Pull request by shankha013k267**
I added an image for HinglishEval in the img/projects, added HinglishEval in projects section, currently the button "Visit page" in HinglishEval project section is not functional. What should be added?

### CODE REVIEW

File: pages/projects/index.html [hunk 1]
```diff
@@ -108,6 +108,24 @@ <h2 class="project-title">SaiMan</h2>
...
         </div>
       </div>
+      <div class="project-box">
+        <!-- Recommended Thumbnail Aspect Ratio 5:3 -->
+        <img class="project-thumbnail" src="./img/projects/project-HinglishEval.png">
+        <div class="project-info">
+          <h2 class="project-title">HinglishEval</h2>
+          <p> Hinglish is a blend of Hindi and English, with frequent usage of English words in sentences with standard Hindi grammar. The HinglishEval project evaluates the effectiveness of code-generation models on Hinglish prompts.  </p>
+          <div class="project-buttons">
+            <a class="project-button" href="https://sathvik040105.github.io/crypto-SaiMan/">
```
Comment (mrigankpawagi) [inline pages/projects/index.html:118]: You can link https://github.com/mrigankpawagi/HinglishEval/blob/main/draft/HinglishEval.pdf for now. We will update this with the Springer link later.

File: pages/projects/index.html [hunk 2]
```diff
@@ -108,6 +108,24 @@ <h2 class="project-title">SaiMan</h2>
           </div>
         </div>
       </div>
+      <div class="project-box">
+        <!-- Recommended Thumbnail Aspect Ratio 5:3 -->
+        <img class="project-thumbnail" src="./img/projects/project-HinglishEval.png">
+        <div class="project-info">
+          <h2 class="project-title">HinglishEval</h2>
+          <p> Hinglish is a blend of Hindi and English, with frequent usage of English words in sentences with standard Hindi grammar. The HinglishEval project evaluates the effectiveness of code-generation models on Hinglish prompts.  </p>
```
Comment (mrigankpawagi) [inline pages/projects/index.html:116]: Can say something like "HinglishEval is a benchmark for evaluating the effectiveness of code-generation models on Hinglish prompts. HinglishEval was presented at COMPUTE 2024."

File: pages/projects/index.html [hunk 3]
```diff
@@ -108,6 +108,24 @@ <h2 class="project-title">SaiMan</h2>
...
+          <div class="project-buttons">
+            <a class="project-button" href="https://sathvik040105.github.io/crypto-SaiMan/">
+              <span class="material-symbols-outlined">open_in_new</span>
+              <span class="project-button-label"> Visit Page </span>
```
Comment (mrigankpawagi) [inline pages/projects/index.html:120]: "View Paper"

File: img/projects/project-HinglishEval.png
Comment (mrigankpawagi) [inline img/projects/project-HinglishEval.png:1]: Haha, I will leave this photo to your taste. We don't have an official graphic for this, although possibly one of the figures from the paper/presentation can be used. @AnirudhG07

**Review by mrigankpawagi — CHANGES_REQUESTED** [review]
Thanks for opening this PR. Would be nice to populate the projects page in order to make it easy to get more funds for the club in future :)

File: pages/projects/index.html (hunk 1 above)
Comment (shankha013k267) [reply pages/projects/index.html]: fixed, and made the other changes too

**Review by mrigankpawagi — APPROVED** [review]
Let us merge for now. Can modify later. Thanks again.

**Pull request merged**